Header header
geometry_msgs/Pose pose
int32 reward
sensor_msgs/Image rgb
sensor_msgs/Image depth